docs/en/ingest-management/elastic-agent/running-on-eks-managed-by-fleet.asciidoc

```diff
@@ -0,0 +1,176 @@
+[[running-on-eks-managed-by-fleet]]
+= Run {agent} on Amazon EKS managed by {fleet}
+
+Use {agent} https://www.docker.elastic.co/r/beats/elastic-agent[Docker images] on Kubernetes to
+retrieve cluster metrics.
+
+TIP: Running {ecloud} on Kubernetes? Refer to {eck-ref}/k8s-elastic-agent-fleet.html[Run {elastic-agent} on ECK].
+
+ifeval::["{release-state}"=="unreleased"]
+
+However, version {version} of {agent} has not yet been
+released, so no Docker image is currently available for this version.
+
+endif::[]
+
+[discrete]
+== Important Notes:
+
+On managed Kubernetes solutions like EKS, {agent} has no access to several data sources. Find below the list of the non available data:
+
+1. Metrics from https://kubernetes.io/docs/concepts/overview/components/#control-plane-components[Kubernetes control plane]
+components are not available. Consequently metrics are not available for `kube-scheduler` and `kube-controller-manager` components.
+In this regard, the respective **dashboards** will not be populated with data.
+2. **Audit logs** are available only on Kubernetes master nodes as well, hence cannot be collected by {agent}.
+3. On Amazon EKS `orchestrator.cluster.name` field is not populated. This field is used as a cluster selector for default Kubernetes dashboards, shipped with https://docs.elastic.co/en/integrations/kubernetes[Kubernetes integration].
++
+In this regard, you can use https://www.elastic.co/guide/en/beats/filebeat/current/add-fields.html[`add_fields` processor] to set `orchestrator.cluster.name` and `orchestrator.cluster.url` fields in order to add those fields for each https://docs.elastic.co/en/integrations/kubernetes[Kubernetes integration]'s component:
++
+[source,yaml]
+.Processors configuration
+------------------------------------------------
+  - add_fields:
```

Comment:
Just make sure the indentation is correct here. If there is an extra level of indentation this might cause problems if users copy/paste it.

Reply:
fixed - https://github.com/elastic/observability-docs/pull/2360/commits/8373a54c6aded40943e8a36f2c659020e211aff0